ee/api/test/test_capture.py

```diff
@@ -20,7 +19,7 @@ def mocked_get_ingest_context_from_token(_: Any) -> None:
 class TestCaptureAPI(APIBaseTest):
     def setUp(self):
         super().setUp()
-        self.client = Client()
+        self.client = Client(enforce_csrf_checks=True)
```

Comment:
Why `enforce_csrf_checks=True`?

Reply:
We hit this in the past. The django test client does some dark magic to avoid CSRF checks which is fine for most uses but when it's really important that CSRF is or isn't applied gives you false confidence.

We had an outage where I'd added a decorator to capture which stopped the CSRF checking from being ignored and took down capture. Adding that param to the constructor of the test client meant the tests immediately showed the problem.

I guess it's a shame that we then have an implicit test because that constructor param is or isn't present when creating the test client